README.md

```diff
@@ -73,6 +73,8 @@ Requirements:
 
    2. Go to where you put LLVM-Tracer source code
 
+       **Skip this step if you want to use CMake to build.**
```

Comment:
Can we add a bit more details here? Something like "If you plan to use the CMake to build Aladdin, we provide an automatic way of installing LLVM and LLVM-Tracer through CMake scripts. In this case, you don't need to install/build LLVM-Tracer separately.